.github/workflows/unit-test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: make install
 
       - name: "Test raco fmt compliance"
-        run: (find . -type f -name '*.rkt' | xargs raco fmt -i) && git diff --exit-code
+        run: raco fmt -i $(find . -name '*.rkt') && git diff --exit-code
```

Comment:
I think this can just run `make format`?

Reply:
Yep